h/static/styles/group-form.scss

```diff
@@ -81,7 +81,7 @@
   }
 
   .group-form-document-list li {
-    margin-top: .5em;
+    margin-top: 1em;
```

Comment:
@robertknight In this instance I measured the image that @conordelahunty gave me (measuring space between two lines of text in an image file - not an exact science) and it came out to 14px which happens to be exactly 1em.